app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/components/SampleViewerTopAppBar.kt

```diff
@@ -36,7 +34,7 @@ fun SampleViewerTopAppBar(
         colors = TopAppBarDefaults.topAppBarColors(
             containerColor = MaterialTheme.colorScheme.primary,
         ),
-        scrollBehavior = scrollBehavior,
+        scrollBehavior = TopAppBarDefaults.exitUntilCollapsedScrollBehavior(rememberTopAppBarState()),
```

Comment:
I have opted to remove this behavior entirely from the Sample Viewer in my audit as it is still an experimental API and noticeably [buggy](https://issuetracker.google.com/issues/268360470)